src/components/List/ListItem.tsx

```diff
@@ -75,7 +87,17 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     accessibility: listItemBehavior as Accessibility,
   }
 
-  state: any = {}
+  constructor(props: IListItemProps, state: IListItemState) {
```

Comment:
there is no `state` passed to component constructor, second arg should be removed

Reply:
fixed